src/transformers/modeling_tf_utils.py

```diff
@@ -1129,6 +1129,119 @@ def call(self, x):
         return x
 
 
+class WordEmbeddings(tf.keras.layers.Layer):
+    def __init__(self, vocab_size: int, hidden_size: int, initializer_range: float, **kwargs):
+        super().__init__(**kwargs)
+
+        self.vocab_size = vocab_size
+        self.hidden_size = hidden_size
+        self.initializer_range = initializer_range
+
+    def build(self, input_shape):
+        self.word_embeddings = self.add_weight(
+            name="weight",
+            shape=[self.vocab_size, self.hidden_size],
+            initializer=get_initializer(initializer_range=self.initializer_range),
+        )
+
+        super().build(input_shape=input_shape)
+
+    def get_config(self):
```

Comment:
What is this function needed for?

Reply:
This is a required function when a layer takes some parameters in its `__init__` see more detail [here](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer#get_config

Reply:
Basically this is what does the @keras_serializable